### PR TITLE
Add production configs

### PR DESCRIPTION
**Pull request by IsaackMwenda**
This adds a couple of production configurations:
1. Adds Extra Kalobeyei demog datasets
2. removes unused congo_swahili language code-  This is not going to be used.
3. updates leap_kakuma_ -> leap_kalobeyei_ in ws correction scheme
4. Adds pipeline config file

### CODE REVIEW

File: code_schemes/kalobeyei_age_of_parent.json [hunk 1]
```diff
@@ -0,0 +1,215 @@
+{
+  "SchemeID": "Scheme-c829a2f1",
+  "Name": "kalobeyei age of parent",
```
Comment (as2388) [inline code_schemes/kalobeyei_age_of_parent.json:3]: Nit:
```suggestion
  "Name": "Kalobeyei Age of Parent",
```
etc

File: code_schemes/kalobeyei_s01e04.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "SchemeID": "Scheme-4a1c98e1",
-  "Name": "KAKUMA S01E04",
+  "SchemeID": "Scheme-fcfbd1f8",
+  "Name": "KALOBEYEI S01E04",
```
Comment (as2388) [inline code_schemes/kalobeyei_s01e04.json:3]: Does e05 need to be renamed/re-id'd? The scheme is showing as renamed without changes for me, whereas all the other files have content changes.

**Review by as2388 — APPROVED** [review]
sorry I missed this earlier

File: code_schemes/kalobeyei_age_of_parent.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/kalobeyei_age_of_parent.json:3]: updated thanks.

File: code_schemes/kalobeyei_s01e04.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/kalobeyei_s01e04.json:3]: Thanks for catching this, have renamed the scheme name for e05 too.

**Pull request merged**